pkg/kubelet/dockertools/manager.go

```diff
@@ -551,9 +554,10 @@ func (dm *DockerManager) runContainer(
 		PidMode:        pidMode,
 		ReadonlyRootfs: readOnlyRootFilesystem(container),
 		// Memory and CPU are set here for newer versions of Docker (1.6+).
-		Memory:     memoryLimit,
-		MemorySwap: -1,
-		CPUShares:  cpuShares,
+		Memory:      memoryLimit,
+		MemorySwap:  -1,
+		CPUShares:   cpuShares,
+		OomScoreAdj: oomScoreAdj,
```

Comment:
Should you check docker API version to decide if OomScoreAdj is allowed? I don't think docker has forward compatibity with old docker API.